src/main/flight/pid.c

```diff
@@ -144,6 +153,7 @@ void resetPidProfile(pidProfile_t *pidProfile)
         .itermLimit = 150,
         .throttle_boost = 5,
         .throttle_boost_cutoff = 15,
+        .anti_gravity_new = true,
```

Comment:
Since `anti_gravity_new` is not appended at the end of PG, let's synchronize the order of initializers as well, moving this line to the end.